### PR TITLE
fix(import): fixed stack  inaccessible issue

### PR DESCRIPTION
**Pull request by vkalta**

### CODE REVIEW

**Review by netrajpatel — DISMISSED** [review]

File: packages/contentstack-import/src/import/module-importer.ts [hunk 1]
```diff
@@ -37,8 +37,8 @@ class ModuleImporter {
       },
     });
```
[the remark this answers is not in the document]
Comment (vkalta) [reply packages/contentstack-import/src/import/module-importer.ts:39]: Its done @netrajpatel .

**Review by netrajpatel — APPROVED** [review]

**Comment by abhinav-from-contentstack** [comment]
Marking this PR as draft, as this is still `in progress` in CS-42157

File: packages/contentstack-import/src/import/module-importer.ts [hunk 2]
```diff
@@ -26,19 +26,21 @@ class ModuleImporter {
...
-    const { data } = await httpClient.post(`https://${this.importConfig.host}/v3/locales`, {
-      locale: {
-        name: 'English',
-        code: 'en-us',
-      },
-    });
-
-    if (data.error_code === 161) {
-      throw new Error(data.error_message);
+    if (this.importConfig.management_token) {
```
Comment (shafeeqd959) [inline packages/contentstack-import/src/import/module-importer.ts:29]: @vkalta , pls move this entire code to a utility function. This effects the code readability

File: packages/contentstack-import/package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@contentstack/cli-cm-import",
   "description": "Contentstack CLI plugin to import content into stack",
-  "version": "1.11.0",
+  "version": "1.11.1",
```
Comment (shafeeqd959) [inline packages/contentstack-import/package.json:4]: Pls verify the version before merging it into development branch

**Review by shafeeqd959 — APPROVED** [review]

File: packages/contentstack-import/package.json (hunk 3 above)
Comment (netrajpatel) [reply packages/contentstack-import/package.json:4]: This version bump is good.

**Pull request merged**